app/assets/stylesheets/participation.scss

```diff
@@ -335,6 +335,12 @@
     word-wrap: break-word;
   }
 
+  .callout.token-message {
```

Comment:
I use only `.callout` class, without blue color. But if you prefer use this one change it to:

```css
.callout {

   &.token-message {
        background-color: #fff;
        border-color: $info-border;
        color: $color-info;
     }
}
```